main.py

```diff
@@ -21,23 +22,25 @@
 }
 
 
-def load_json(json_file, lib_to_use):
+def load_file(json_file, lib_to_use):
```

Comment:
we should probably rename json_file here to something like input_file since we are parsing yaml now too.